.github/workflows/client.yaml

```diff
@@ -29,6 +29,9 @@ jobs:
         with:
           node-version: "16"
 
+      - name: Install Playwright Dependencies
+        run: npx playwright install-deps chromium
```

Comment:
Where is Chromium stored? Is it kept in the PNPM cache (above)?

Reply:
This installs the Chromium dependencies through the normal apt-get-whatever package manager, so it's unfortunately slow and is redone everytime. I don't know if there is a way to avoid this.

Reply:
Whatever `/usr` directory it's installed to could be cached. Or actually probably you should cache the APT package cache dir.

Reply:
Okay, I gave it a shot. I didn't try directly caching anything, because when I did some googling all I found was discouraging results. It seems caching installs like this is ill-advised. I tried using a container someone had made for Playwright, hoping that GitHub would maybe do some persistent Docker magic on it or something, but it was too messy and I don't think it worked anyways.

If you wanna give this a shot, go ahead, but for right now it's okay as it is.

Reply:
Oh well, it was worth looking into at least.